chat/tasks.py

```diff
@@ -45,9 +49,18 @@ def cleanup_puzzle_channels(puzzle_id):
                 logger.warn(f"cleanup_puzzle_channels failed with error: {e}")
 
 
+@shared_task
+def handle_puzzle_meta_change(puzzle_id):
```

Comment:
Looks like this is called in two cases: (1) the set of this puzzle's metas has changed, (2) this puzzle itself has toggled its `is_meta` state. I think the name is not totally obvious, maybe we should add a docstring here? Or split into two methods that do the same thing (maybe not worth it, not sure).